src/scss/components/checkout/paymentProvider/_paymentProvider.scss

```diff
@@ -79,3 +79,8 @@ body.klarna-payments-fso-open {
 #stripe-card-field--webkit-autofill {
     background-color: #fefde5 !important;
 }
+
+#paymentButtonWidget {
```

Comment:
This is also no longer needed.

Reply:
I renamed it to  class `submitButtonContainer`